src/tx_buffers.rs

```diff
@@ -221,24 +222,34 @@ impl<'a, P: crate::CanId, C: Capacities> Tx<'a, P, C> {
         self.get_transmission_completed_flags().iter()
     }
 
-    /// Request cancellation of `buffers`. This function does not wait for
-    /// cancellation to finish. Successful cancellation is indicated by a
-    /// combination of the transmission completed and cancellation flags. If
-    /// the flags indicate that transmission is completed, then the request
-    /// arrived too late to stop the transmission, which was completed
-    /// successfully. If the cancellation flag is set, but not the
-    /// transmission completed flag, the transmission was either not started or
-    /// was aborted due to an error.
-    pub fn cancel_multi(&mut self, buffers: TxBufferSet) {
-        // Safety: There are no reserved bit patterns.
-        unsafe {
-            self.txbcr().write(|w| w.bits(buffers.0));
+    fn poll_canceled(&self, buffers: TxBufferSet) -> nb::Result<(), Infallible> {
```

Comment:
Maybe instead of `buffers` something slightly more descriptive. Like `to_be_canceled`, so the comparison with the mask is slightly more readable.

Reply:
All right. I changed the other functions that follow the same pattern as well to remain consistent.